src/Common/tests/System/Diagnostics/AssertWithCallerAttributes.cs

```diff
@@ -115,6 +115,20 @@ internal static class Assert
         catch (Exception e) { throw WrapException(e, path, line); }
     }
 
+    public static void DoesNotContain(IEnumerable collection, object expected,
+        [CallerFilePath] string path = null, [CallerLineNumber] int line = 0)
+    {
+        try { Xunit.Assert.DoesNotContain(expected, (IEnumerable<object>)collection); }
+        catch (Exception e) { throw WrapException(e, path, line); }
+    }
+
+    public static void DoesNotContain<T>(IEnumerable<T> collection, T expected,
```

Comment:
I think there may have been a couple of "Assert.None" overloads that had parameters in different orders. I deleted the ones that didn't directly wrap DoesNotContain calls.